sktime/dists_kernels/compose_tab_to_panel.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+Composers that create panel pairwise transformers from table pairwise transformers
+"""
+
+__author__ = ["fkiraly"]
+
+import numpy as np
+
+from sktime.dists_kernels._base import BasePairwiseTransformerPanel
+
+
+class AggrDist(BasePairwiseTransformerPanel):
+    """
+    pairwise distance based on aggregation function applied to sample-wise distance
+        example: AggrDist(ScipyDist()) is mean Euclidean distance between series
+
+    Components
+    ----------
+    transformer: pairwise transformer of BaseTrafoPw scitype (tabular pairwise)
+
+    Hyper-parameters
+    ----------------
+
+    aggfunc: aggregation function 2D np.array -> float
+        default = np.mean
+    """
+
+    def __init__(
+        self,
+        transformer,
+        aggfunc=None,
```

Comment:
Minor comment:
What is the reason reason why default is None instead of np.mean?